engine/runtime/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -107,28 +108,14 @@ class Compiler(
       }
 
       if (irCachingEnabled && !builtins.getModule.wasLoadedFromCache()) {
-        serializationManager.serialize(
+        serializationManager.serializeModule(
           builtins.getModule,
           useGlobalCacheLocations = true // Builtins can't have a local cache.
```

Comment:
Interesting (but unrelated) comment. Why builtins can't have a local cache? If "local" means inside of the (standard) library which is inside of an Enso distribution (which also contains implementation of all the builtins), then I believe the caches should be there, ready for the 1st usage.